tests/coafileTest.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+import re
+import os
+from coalib.misc.Shell import run_shell_command
+from coalib.settings.ConfigurationGathering import load_config_file
+
+
+class coafileTest(unittest.TestCase):
+
+    def setUp(self):
+        self.filepath = '.coafile'
+        self.assertTrue(os.path.isfile(self.filepath),
```

Comment:
@Mixih the logprinter in load_config_file was only used for filenotfound exception.So i dont think it is necessary to pass a logprinter at all.It futureproofs it too.

yeah.. I think i overdid it in checking if the file exist though :)